doc/api/stream.md

```diff
@@ -2046,6 +2057,18 @@ added: REPLACEME
   * `signal` {AbortSignal}
 * Returns: {stream.Readable}
 
+### `stream.Readable.isDisturbed(stream)`
```

Comment:
I think the name here could be more expressive, maybe `.hasEmittedStreamEvents()`? I know it's generic, but so is "disturbed", and it's a bit more precise.

We should also add to the documentation that this only starts being `true` when data has been *emitted* (i.e. `stream.on('data')`, not when data has been requested from the underlying resource (i.e. `stream._read()`) or made available to the stream implementation (i.e. `stream.push()`).

Reply:
> I think the name here could be more expressive, maybe .hasEmittedStreamEvents()? I know it's generic, but so is "disturbed", and it's a bit more precise.

I disagree. It's not just when it has been read, it's also when it has been cancelled. The WHATWG spec calls this state `disturbed`. Why make up a different name?

> We should also add to the documentation that this only starts being true when data has been emitted (i.e. stream.on('data'), not when data has been requested from the underlying resource (i.e. stream._read()) or made available to the stream implementation (i.e. stream.push()).

I think that it is kind of clear. Neither `push` nor `_read` "reads" from the `Readable` interface. Anyway I'm open for suggestions...

Reply:
>  Why make up a different name?

The argument is: Because the spec naming isn't typically meant for end-users and streams are very confusing to most of our users anyway basically.